controllers/articles.controller.js

```diff
@@ -19,3 +19,29 @@ exports.getAllArticles = (req, res, next) => {
     .catch(next)
 };
 
+// exports.postCommentByArticle = (req, res, next) => {
+//     console.log(req.body);
+//     const { article_id } = req.params;
+//     const { username, body } = req.body;
+//     console.log(article_id, username, body);
+//     insertComment(article_id, username, body)
+//         .then((comment) => {
+//             res.status(201).send({ comment });
+//         })
+//         .catch(next);
+// };
+
+exports.postCommentByArticle = (req, res, next) => {
+    const { article_id } = req.params;
+    const { username, body } = req.body;
+    if (!username || !body) {
```

Comment:
If you look at the seed file, you'll see there is a not null constraint on both the username and body. This means if they're missing, psql will kick out an error code. You shouldn't need a custom error handler here if you've set up your psql error handling functions.